contracts/Unibright.sol

```diff
@@ -67,13 +67,20 @@ contract UBTSplitter is Context, Ownable {
     uint256 public state_lastEventNonce;
```

Comment:
just a reminder to fix all these warnings
also, we can remove _pendingPayment function